Migrate backend-requests 'Overview' from Sanity to MDX (#288)
clerkinc/clerk-docs: docs/backend-requests/overview.mdx

---
title: Request authentication

> description: Learn about various ways to make authenticated requests to the backend using the Clerk SDK.

Reviewer: ```suggestion
description: Learn about various ways to make authenticated requests to the backend using Clerk's SDK.
```

---

# Request authentication

A request is considered “authenticated” when the backend can securely identify which user and which device is making the request. The reasons for making authenticated requests to the backend include:

- Associating the user with the action being performed
- Ensuring the user has permission to make the request
- Keeping an audit log of which device the user is performing actions from

> In order to authenticate the user on the backend using the Clerk SDK, the short-lived [session token](/docs/backend-requests/resources/session-tokens) needs to be passed to the server.

Reviewer: ```suggestion
In order to authenticate the user on the backend using Clerk's SDK, the short-lived [session token](/docs/backend-requests/resources/session-tokens) needs to be passed to the server.
```

## Frontend requests

To make authenticated requests from the frontend, the approach differs based on whether your client and server are on the same origin. 

The origin includes the protocol, hostname, and port (optional):

`<protocol>//<hostname>[:<port>]`

### Same-origin

For same-origin requests, refer to our guide on [making same-origin requests](/docs/backend-requests/making/same-origin).

### Cross-origin

For cross-origin requests, refer to our guide on [making cross-origin requests](/docs/backend-requests/making/cross-origin).

## Backend requests

> Clerk provides various middleware packages to set the session property for easy access and can also require a session to be available. Choose the guide based on the language or framework you're using:

Reviewer: ```suggestion
Clerk provides various middleware packages to set the session property for easy access. These packages can also require a session to be available on the current request. Choose the guide based on the language or framework you're using:
```
